en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -84,15 +84,13 @@ Follow the steps below to learn how to drill down and confirm the root cause for
         The time range selected for the **Throughput & Latency** graphs is applied to the **Diagnostics View** by default. We recommend capturing the time range you want to analyze in detail from the **Throughput & Latency** graph and then navigate to the **Diagnostics View**.
         
 2. In the sample data, you can observe a couple of prominent errors appearing in the four bins.
-3. Select a bin that shows a higher occurrence of the error log "error while connecting to the hr-service”. Notice that this bin also includes other errors which occurred at a lesser frequency.
+3. Select a bin that shows a higher occurrence of the error log "error while connecting to the hr-service”. Notice that this bin may include other errors which occurred at a lesser frequency.
 
-    ![Diagnostic View - second bin](../assets/img/observability/second-bin.png){.cInlineImage-full}
+    ![Diagnostic View - second bin](../assets/img/observability/second-bin.png){.cInlineImage-bordered}
```

Comment:
Seems like the details in the log contradicts with what is mentioned in step 3 right?